### ISSUE TITLE
Liquidation Alert

### ISSUE THREAD

**Issue by oksuzali999** [Tucsky/aggr#315]
First of all, thank you for this beautiful work.
Can an alarm be added to liquidation rates as well as prices? For example, in BTC, can I create a warning with an alarm when there is a liquidation of 250 k up or down? How should I go about doing this.

**Comment by adeacetis** [comment]
Hello, 

It's impossible to set an Alert to something different than the price now. However, I reckon that it would be indeed useful. I am adding this to the feature backlog.

FYI @Tucsky

**Comment by lmvdz** [comment]
https://github.com/lmvdz/aggr/commit/389b603699830ce6e066df4f1464fc428dbb9922

converted alertService to allow for indicators, just need help from @Tucsky to get the value of the indicator instead of prices from aggregator service in the `getValue` function. https://github.com/lmvdz/aggr/commit/389b603699830ce6e066df4f1464fc428dbb9922#diff-aed1a9b5af387801328e5252658e7eb3b99ec2b2f75f1a237fedc4a73ce58935R302

**Comment by lmvdz** [comment]
Looking into this more, and will need more work than just the `alertService.ts`.
- displaying the alert "price" line on the chart will have to be changed to use the indicator's axis.

**Comment by oksuzali999** [comment]
yes, I realized that I need to concentrate on "alertService.ts". Including "lsell" and "lbuy"... I've been trying to achieve this for a long time. This situation requires experience beyond my coding knowledge and I failed to do so.

**Comment by raid5** [comment]
I'm glad to see this being added to the feature backlog, I came here to request the same thing! I would _**LOVE**_ volume alerts.

Also, maybe close https://github.com/Tucsky/aggr/issues/238 as a dupe since this issue has more context now?

**Comment by mhakby** [comment]
Any progress on this? If not, can you tell me when it might happen?

**Comment by Tucsky** [comment]
@mhakby I wouldn't get my hopes up unless someone explicitly said that hes working on it @mhakby

`alertService` mentioned above is exclusively for server side price alerts, alerts are then grouped by indexes (avg price of the coin accross the different contracts) and range of prices for saving performance which is incompatible with any other type of value.

Only way I can think of, is for you to fork this Github project and edit the Prices.vue pane (as it is the closest thing to a screener)
while making use of `this.$store.dispatch('app/showNotice', ...)` and `audioService.playOnce(your alert sound name)` to get the info while the app is open

Staying available here or on discord if someone wants to give it a go and try to implement it. Also PR is very much appreciated, as always :)

**Comment by mrjamesbruno** [comment]
Hi, is it currently possible to get "notification helper function in indicator script so you can get notifications"? Any news about it?
If yes, please point me to where I can read how to achieve this.
Thanks
